### DISCUSSION TITLE
OCI-Build task

### DISCUSSION THREAD

**Discussion by prime89-hash** [concourse/concourse#7867] [Help & Support] [answered]
Hi @taylorsilva , 
We want to setup OCI-BUILD-TASK to our concoursev7 configuration, could you please help us to understand how we can configure that into our web configuration.

**Comment by taylorsilva** [comment, marked as answer]
There's nothing to configure on the web node in order to use the oci-build task. 

These guides show you how to use the oci-build task: https://concourse-ci.org/container-image-guides.html

**Reply by prime89-hash** [reply]
Thank you so much @taylorsilva for the information.
